src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i <= 5; i++) {
+            if (i <= 2) {
```

Comment:
Instead of using hardcoded numbers, consider using a variable to define the number of figures to make an easy change if we require another amount of figures
```suggestion
        Figure[] figures = new Figure[FIGURES_NUMBER];
        for (int i = 0; i < FIGURES_NUMBER; i++) {
            if (i <= FIGURES_NUMBER / 2) {
```